## Description: Fixed frecquency value step in manual chapter "Usage/Frequency".
## Origin: upstream, no
## Bug-Ubuntu: http://launchpad.net/bugs/980652
## Bug-Debian: 
## Author: Pojar George
## Forwarded: no
Index: b/help/C/gnomeradio.xml
===================================================================
--- a/help/C/gnomeradio.xml	2012-12-03 10:14:55.306899796 +0000
+++ b/help/C/gnomeradio.xml	2012-12-03 10:14:55.666899785 +0000
@@ -171,12 +171,12 @@
     </listitem>
     <listitem>
         <para>
-	   The <guibutton>0.5 MHz backwards</guibutton> button (left-arrow without bar) decreases the frequency by 0.5 MHz.
+	   The <guibutton>0.05 MHz backwards</guibutton> button (left-arrow without bar) decreases the frequency by 0.05 MHz.
         </para>
     </listitem>
     <listitem>
         <para>
-	   The <guibutton>0.5 MHz forward</guibutton> button (right-arrow without bar) increases the frequency by 0.5 MHz.
+	   The <guibutton>0.05 MHz forward</guibutton> button (right-arrow without bar) increases the frequency by 0.05 MHz.
         </para>
     </listitem>
     <listitem>
